<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QDialog" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>984</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView {
	border: none;
	background: transparent;
}

QHeaderView::section
{
	spacing: 10px;
	background-color: gray;
	color: white;
	border: 1px solid ;
	margin: 1px;
	text-align: right;
	font-family: arial;
	font-size:12px;
}

QPushButton {
	background: lightgray;
	border-radius: 5px;
	font-size: 12px;
	height: 35px;
}

QPushButton#pushButton_back {
	width: 85px;
}

QPushButton#pushButton_back:hover {
	background: lightblue;
}

QPushButton:hover {
	background: darkgray;
	color: white;
	font-weight: bold;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1288</width>
        <height>2169</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_back">
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="Label">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Admin </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>9999</width>
             <height>9999</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="res.qrc">:/Res/Pics/admin.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <property name="bottomMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QPushButton" name="admin_showNFLDistances">
             <property name="text">
              <string>Show NFL Distances</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="admin_showNFLSouvenirs">
             <property name="text">
              <string>Show NFL Souvenirs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="admin_showNFLInfo">
             <property name="text">
              <string>Show NFL Teams</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_extras">
             <property name="text">
              <string>Admin Extras</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="admin_commitChanges">
           <property name="text">
            <string>Commit Change(s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="admin_addSailors">
           <property name="text">
            <string>Add Sailors</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="admin_moveRams">
           <property name="text">
            <string>Move Rams Location</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="admin_addSouvenir">
           <property name="text">
            <string>Add Souvenir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="admin_deleteSouvenir">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Delete Souvenir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="admin_addSouvFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="5" column="2">
             <widget class="QPushButton" name="admin_addSouvenirFinal">
              <property name="text">
               <string>Add Souvenir</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Choose a stadium:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Add Souvenir</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="admin_newSouvName">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhDate</set>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>New Souvenir Name:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QComboBox" name="admin_addSouvCombo">
              <item>
               <property name="text">
                <string>AT&amp;T Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Arrowhead Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bank of America Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CenturyLink Field</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>EverBank Field</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>FedExField</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>FirstEnergy Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ford Field</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Georgia Dome</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Gillette Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hard Rock Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Heinz Field</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lambeau Field</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Levi's Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lincoln Financial Field</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Los Angeles Memorial Coliseum</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lucas Oil Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>M&amp;T Bank Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Mercedes-Benz Superdome</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MetLife Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NRG Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Nissan Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Oakland Alameda Coliseum</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Paul Brown Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qualcomm Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ralph Wilson Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Raymond James Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Soldier Field</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sports Authority Field at Mile High</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Stub Hub Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>U.S. Bank Stadium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>University of Phoenix Stadium</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QDoubleSpinBox" name="admin_newSouvPrice">
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>9999.989999999999782</double>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>New Souvenir Price:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="admin_labelSearch">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>7</pointsize>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>Search:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="admin_searchBar">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableView" name="admin_tableview">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>1000</height>
              </size>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
